channels/hk.m3u

```diff
@@ -29,3 +29,17 @@ http://221.179.217.70/PLTV/88888888/224/3221225942/1.m3u8
 http://117.169.120.138:8080/live/fhzixun/.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Chinese" tvg-logo="https://p1.ifengimg.com/a/2018_43/45a9d426b4ae58c.jpg" group-title="",鳳凰衛視電影台
 rtmp://ivi.bupt.edu.cn:1935/livetv/fhdy
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Chinese;Yue Chinese" tvg-logo="http://img.tvb.com/corporate/_fck_/image/Jade_logo_L(1).jpg" group-title="",翡翠台（81）
+https://cdn.hkdtmb.com/hls/81/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Chinese;Yue Chinese" tvg-logo="http://img.tvb.com/corporate/_fck_/image/Jade_logo_L(1).jpg" group-title="",翡翠台（81）
+http://61.238.6.49:8000/bysid/1
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Chinese;Yue Chinese" tvg-logo="http://img.tvb.com/corporate/_fck_/image/J2_logo_L(2)" group-title="",J2（82）
+http://61.238.6.49:8001/bysid/82
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Chinese;Yue Chinese" tvg-logo="http://img.tvb.com/corporate/_fck_/image/NEWS2017_logo_L.jpg" group-title="",無綫新聞資訊台（83）
+http://61.238.6.49:8001/bysid/83
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="http://img.tvb.com/corporate/_fck_/image/Pearl_E_logo_L.jpg" group-title="",Pearl（84）
+http://61.238.6.49:8000/bysid/2
```

Comment:
Can you check this link? I can't get it to work.
The rest is good.

Reply:
> Can you check this link? I can't get it to work.
> The rest is good.

[img]https://i.imgur.com/HGB4Y9R.png[/img]
just checked, the 83 link is working

but the other one was gone, hmm